lib/orogen/spec/project.rb

```diff
@@ -38,8 +41,13 @@ def initialize(loader)
                 @define_default_deployments = true
                 @enabled_transports = Set.new
                 @max_sizes = Hash.new
+                @disabled_namespaces = ['test']
```

Comment:
That would be wrong. While it does make sense in gen, it is IMO harmful in spec. In any case, it is broken w.r.t. the code generated in orogen as you only call disable_namespace (and therefore have no way to enable the test namespace)